Blue_Green_Ami.pkr.hcl

```diff
@@ -0,0 +1,89 @@
+packer {
+  required_plugins {
+    amazon = {
+      version = ">= 1.1.1"
+      source  = "github.com/hashicorp/amazon"
+    }
+  }
+}
+
+variable "ami_prefix1" {
+  type    = string
+  default = "Blue_Ami"
+}
+
+locals {
+  timestamp = regex_replace(timestamp(), "[- TZ:]", "")
+}
+
+source "amazon-ebs" "ubuntu" {
```

Comment:
Could you provide a proper name for the resource here please, instead of just the `ubuntu` name. E.g. this is for the blue ami, so you could try a name like `blue-web-server`.